fvm/src/gas/price_list.rs

```diff
@@ -102,19 +101,14 @@ lazy_static! {
         address_lookup: Gas::new(1_050_000),
         address_assignment: Gas::new(1_000_000),
 
-        sig_cost: total_enum_map!{
-            SignatureType {
-                Secp256k1 => ScalingCost {
-                    flat: Gas::new(1637292),
-                    scale: Gas::new(10),
-                },
-                BLS =>  ScalingCost{
-                    flat: Gas::new(16598605),
-                    scale: Gas::new(26),
-                },
-            }
-        },
         secp256k1_recover_cost: Gas::new(1637292),
+        bls_pairing_cost: Gas::new(8299302),
+        bls_hashing_cost: ScalingCost {
+            flat: Gas::zero(),
+            // TODO: Can we just use the "sha256" number below?
+            scale: Gas::new(7),
```

Comment:
Just a note: this PR did not change the BLS signature hash function (that maps a plaintext's bytes to a curve point). I'm guessing that this gas price change (to that of SHA256's) was intentional as SHA256 is used internally by the BLS hash function.

Reply:
Yeah, that makes sense. I'll leave it as its own separate price just to be safe.